docs/create/homebrew/index-homebrew.mdx

```diff
@@ -1,11 +1,28 @@
 ---
 id: index-homebrew
-title: Create Redis database on Mac 
-sidebar_label: Redis on Mac
+title: Create Redis database on Mac OS 
```

Comment:
```suggestion
title: Create a Redis database on macOS 
```

Reply:
The space between Mac and OS has been added intentionally for SEO purpose.